screens/controls.py

```diff
@@ -89,9 +90,9 @@ def toggle_full_screen():
 def controls():
     run = True
 
-    control_title_font = pygame.font.Font(os.path.join(FONT_PATH, 'edit_undo.ttf'), 50)
-    control_font = pygame.font.Font(os.path.join(FONT_PATH, 'neue.ttf'), 30)
-    keys_font = pygame.font.Font(os.path.join(FONT_PATH, 'neue.ttf'), 30)
+    control_title_font = pygame.font.Font(resource_path(os.path.join(FONT_PATH, 'edit_undo.ttf')), 50)
+    control_font = pygame.font.Font(resource_path(os.path.join(FONT_PATH, 'edit_undo.ttf')), 30)
+    keys_font = pygame.font.Font(resource_path(os.path.join(FONT_PATH, 'edit_undo.ttf')), 30)
```

Comment:
Here I think neue font looks better for now

Reply:
I ran a scripted replace operation so that is how it probably happened. It will be reverted.